docs/source/using/troubleshooting.adoc

```diff
@@ -314,3 +314,28 @@ You can configure the URL by running:
 $ minishift config set check-network-http-host <URL>
 ----
 
+[[github-api-rate-limit-exceeded]]
+== GitHub API rate limit exceeded
+
+When you run `minishift start` or `minishift update`, it might make a request to GitHub to get the latest tarball of the Minishift or OpenShift binary.
```

Comment:
I would say then:

"When you run `minishift start` or `minishift update`, it makes requests to GitHub API to check for new version and potentially download the latest tarball of the Minishift or OpenShift client tool `oc`."